src/hotspot/cpu/ppc/sharedRuntime_ppc.cpp

```diff
@@ -1823,7 +1743,7 @@ nmethod *SharedRuntime::generate_native_wrapper(MacroAssembler *masm,
   //
   // NW     [ABI_REG_ARGS]             <-- 1) R1_SP
   //        [outgoing arguments]       <-- 2) R1_SP + out_arg_slot_offset
-  //        [oopHandle area]           <-- 3) R1_SP + oop_handle_offset (save area for critical natives)
+  //        [oopHandle area]           <-- 3) R1_SP + oop_handle_offset (save area for critical natives) ?
```

Comment:
`?`. The comment `(save area for critical natives)` must be redundant now.

Reply:
I didn't know if the save area is still needed for something else, which is why I left the ?.  I can remove the comment but haven't made any  substantial changes here.  I'm not sure if they're needed or not, but I can't test them if I made them.

Reply:
I mean, you did the same here: https://github.com/openjdk/jdk/pull/6343/files#diff-060e534de775616a893aa969f3639e53666cda9e93bed7c3a3c14b9cdc4cdba0L1553-L1554 -- and that change is understandable.

Reply:
Looking further, the area is needed (stores oops there), but not the comment.